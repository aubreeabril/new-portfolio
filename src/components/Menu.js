import React, { Component } from "react";
import { Navbar, NavItem, Nav, NavbarBrand, NavLink } from "reactstrap";

class Menu extends Component {
  render() {
    return (
      <Navbar sticky="top" className="menu">
        <Nav>
          <NavbarBrand href="/portfolio/#aa">
            <strong>AA</strong>
          </NavbarBrand>
          <NavItem>
            <strong>
              <NavLink href="/portfolio/#about">About</NavLink>
            </strong>
          </NavItem>
          <NavItem>
            <strong>
              <NavLink href="/portfolio/#projects">Projects</NavLink>
            </strong>
          </NavItem>
        </Nav>
      </Navbar>
    );
  }
}

export default Menu;
