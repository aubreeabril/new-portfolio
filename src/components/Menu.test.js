import React from "react";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";
import Menu from "./Menu";

describe("Menu", () => {
  it("renders the menu", () => {
    const wrapper = shallow(<Menu />);
    expect(wrapper.find(".menu").length).toEqual(1);
  });
  it("matches its snapshot", () => {
    const section = renderer.create(<Menu />).toJSON();
    expect(section).toMatchSnapshot();
  });
});
