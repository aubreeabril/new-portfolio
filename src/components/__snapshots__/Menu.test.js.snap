// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu matches its snapshot 1`] = `
<nav
  className="menu navbar sticky-top"
>
  <ul
    className="nav"
  >
    <a
      className="navbar-brand"
      href="/portfolio/#aa"
    >
      <strong>
        AA
      </strong>
    </a>
    <li
      className="nav-item"
    >
      <strong>
        <a
          className="nav-link"
          href="/portfolio/#about"
          onClick={[Function]}
        >
          About
        </a>
      </strong>
    </li>
    <li
      className="nav-item"
    >
      <strong>
        <a
          className="nav-link"
          href="/portfolio/#projects"
          onClick={[Function]}
        >
          Projects
        </a>
      </strong>
    </li>
  </ul>
</nav>
`;
